Vagrantfile

```diff
@@ -66,35 +66,38 @@ Vagrant.configure(2) do |config|
   # documentation for more information about their specific syntax and use.
    config.vm.provision "shell", inline: <<-SHELL
 
-      # Don't do this more than once in an hour
-      if [ -z "$(find /var/cache/apt/pkgcache.bin -mmin -60)" ]; then
-          sudo aptitude update
-          # install 'add-apt-repository'
-          sudo aptitude install -y python-software-properties software-properties-common
+      sudo apt-get install -y aptitude
```

Comment:
I think it would be a bit better to use `apt` instead of `aptitude`. I believe everything below will just work. `aptitude` has mostly been replace by `apt`, which is why it is no longer in the default install. That said, I don't think this should block merging this unless you wanted to make that change

Reply:
I didn't know that.  I'll make this change then.

Reply:
Fixed in 8097b18e0761c